packages/ember-routing-views/lib/views/link.js

```diff
@@ -421,17 +412,6 @@ var LinkComponent = EmberComponent.extend({
       this.set('disabled', attrs.disabledWhen);
     }
 
-    var currentWhen = attrs['current-when'];
-
-    if (attrs.currentWhen) {
-      Ember.deprecate('Using currentWhen with {{link-to}} is deprecated in favor of `current-when`.', !attrs.currentWhen);
-      currentWhen = attrs.currentWhen;
-    }
-
-    if (currentWhen) {
```

Comment:
I considered that, but since the `currentWhen` attribute was deprecated explicitly set and **was marked as private** in the docs I wasn't sure if I needed to go through the intermediate step. Do I?